icons/angular/angular-original-wordmark.svg

```diff
@@ -1,15 +1,39 @@
EHoAm5DvkohDbplR8PkI2H8dADxrXqpafoeSXzGihSilgYGAAAAAA//8aLVipAHj2NzPw7m9mYHHQBhuGWsgSynLoLUx0gLsgxjcuSgogzZz/SBkdV2VBpEkUD6NiH4tFHVbAZ8l/lNiCCWF2kwkNCWB27Ql181GvjUYdp8VMNfhinwF+8DWqPswCGJWGDQmAuv0cDOF7xBgsa/mwuHoUkAwYGBgAAAAA//8aHQqgMvhz4CrDJ8cGHEMCjCjJHXeXHrc8A8aYK+ZBK5jdQcJdd9KGBYjryhM1LIBziAD/kANxQwG4xHF35TH8xMiARS+yHxmwmom/S4+mDkeXHJ95DEjVBlyOwMEr2OjQPeIMMvYcgzErDV3AwMAAAAAA//8abbFSGYBarUL/VzNwNoQS6Kxhb9Fik0cG6AU1NoC9UCWvcYhbD3JrEHvLkeQuPckOJL/Ji9mVx2SjdtsRYoS64AQGQtAGUZBbrvjMwxaaaEUljnWq2IBFnQBD4W/50UKVFoCBgQEAAAD//xptsdIQ/D5wjeF741qG3weugy2BTf5Qr7VKessTmxkImglDnthWLH557HZiiiG11LBMcBHbGsVmJ35xbK1jNPMZsellwOp+bGzsrVW0likjevxgn+RCtR9NDSM2tQhaxp6TwbxWcLRApSVgYGAAAAAA//8aLVjpAEAF6yfHFhyFIgOWAhSfOPauPukFLPaCFlvhStwQADYxBqzqCBfSCH3o57EiChxCKxQoKVjR/Q+x7z+88MNWwGHagz0c0LvwaPZjDAvgMgsPjXGmKwODNLhAFWKQHi1QaQ8YGBgAAAAA//8aHQqgA2B10GQQ/r+UgashCItl+IcLcHenUYcCUDt/yBkLmzx6ZxJhDiMD7noWXw3MiFUdckubFIDkJqy9fUYMWwi5lBG7QSj+xT0kgLAPcTkhdrXYuu2YsYQ91vGFPaZZ6PagC0PkQAVp0B4pMB4tVOkEGBgYAAAAAP//Gm2xDgD41rie4deBGwy/D9zE0opFHUfF1oolrZXKQKQ6wvqIbcViM4s4c/BPWIFprJNJxHTtsbXOCbdWcXXj/yMND6CqZcLZhcfeU8AyUYW2zhWbWYRoaQcuBtNaYXBLdRTQGTAwMAAAAAD//xotWAcQ/D5wg+Fr40aGX0gFLLbCDXthir0Axl9QMpFZ0GIWVJQMCxAumLGrRS18sMvjtodQAUraUAGKWxmJsYMBi/8YMNwH14dlvJWYglbanovBpE5ktEAdSMDAwAAAAAD//xotWAcBABWskAL2FtYCFFuBh68QpHZLFp9awmJEtELJKVhhYhhjsERMRBHTGiWqQEbSz4jdLHx6EGZjijEw4L//ChHPEFrSnovBtE6EQcp+9CT/AQcMDAwAAAAA//8aLVgHEQAVsF8aN4MLWMxClIHKBSvhApbUwha/GQwE1BGSx1NII7VesbsLm9nYu+bEFLxYW7aMDHjMYsARbugFK5ocEasEjOtEwYXpaIE6iAADAwMAAAD//xotWAcp+NK4heFzwxY8hSkD0QUtujj+ApHUFQEMeNVgbWUSKIAJuQ2rPMZ4JyHz0NUSszIAn7kMWGbjMQtDTPsZsIQ9tlUCqHqM68TAheooGISAgYEBAAAA//8aLVgHOfjcuJXhc8NWIgpRBgIFLP7WKfmtWAY8coTVESo4iWqxorcGGVHtxeZmylu0DNj1E9VyxaYGU4wBy3CAEbhAFRuJWWHoAAYGBgAAAAD//xotWIcI+Ny4jeHngTsMPw/cJqqVSlwBiq+FSs5GAUx7qVewEtKPbewV90QUshjh4QN0eTwbBRixzPKT0FrFJmZYL84gac/DIGk/eh3KkAAMDAwAAAAA//8aLViHIPjUuB1ayN4lWChS3kLFLYevsCG5K0+RPI5KAMvSLIIFI0F5IpZjMaK7lfgCFsaWsOdhMKyXANOjYIgBBgYGAAAAAP//Gi1Yhzj41LiT4UPDThyFKP7xUsKFK3H6iWuxEtsSJq4wJGqIAGvBSsI6VTzyxLZcSSlMxe15GSQceBkM6iRGerIe2oCBgQEAAAD//xotWIcJALVefxy8y/ChYTcZ3X5iW6bEDA8Qs+aVAY8ZxE42ESqsoe5gxFHwUalgxdmqJbBUCllMv16aQb9OcqQn4eEDGBgYAAAAAP//Gi1YhyH4ceAew4+D9xg+NOwho4AlLI9fDXGbCXAX5KQWrJjmYKjFcoAK7lYwvhYpCQUrA/7xVr16GXALFYRHwTADDAwMAAAAAP//Gi1Yhzn40LgXnJHfN+wjsYWKKU+8GkLjrYS79eQUovgLVuJbo9jdwYDHThwbANAKVt16WQYxcGE6eqD0sAYMDAwAAAAA//8aLVhHGHjfuA+c0d81HAB7nNguO+7WKy41DFgLR1JbrLi77PgKYyxqGdHFCG1dxbfcirh1qmIOAgxiDvwMOnUyIz3ZjSzAwMAAAAAA//8aLVhHOHjXeABcGLxtOIi3a09KIUj8sABxy6HwF9TEbEMltL4Ue1eeUMsWPQxEHQQYRO35GUQd+BnE7PlHetIauYCBgQEAAAD//xotWEcBCnjbeAhcULxpOExUy5XwZgLCLVBihwIYiCpEGbAWyshDAsS2SAkVrJr1CigF6igYBWDAwMAAAAAA//8aLVhHAV7wuvEIWPrrgcdgTOl4K3EtWgbchSPJ6tDswFhjitmFx9YqF3EQZBCxFwLzNesVRxPNKMANGBgYAAAAAP//Gi1YRwFZ4FXjcbC2/+BC9ynD5wNPSBqrJX0ogJguPO4WJ4pb8Gx5FXIQYhCxFwaLizgIMQjbC44mkFFAGmBgYAAAAAD//xotWEcB1cGLxpPwQut5wxmC3Xhix1OxtSYJ60W1V9BBhEHIQQQqDlGrWq86mghGAfUAAwMDAAAA//8aLVhHwYCCp43n4AUgA9ICegaUghdTDpFoMdUp1muNRuooGDjAwMAAAAAA//8DADWgR0uz5fGBAAAAAElFTkSuQmCC"/>
```

Comment:
This SVG contains a link to a raster image (a PNG), so it doesn't scale correctly. You'll need to find or create a SVG version before it can be accepted into devicons.

Reply:
Didn't think about getting that from the website, thank you!